examples/web3.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback } from 'react';
```

Comment:
should web3 and/or this example be mentioned in the readme? i dont think we have official docs for this lib other than the readme file so it should be comprehensive of the libraries functionality

Reply:
https://github.com/clayallsopp/react-plaid-link/tree/callsopp-web3#usage-with-wallet-onboard viewable